### ISSUE TITLE
KeyError on rescanning subscriptions

### ISSUE THREAD

**Issue by sarthurdev** [bbilly1/tubearchivist#101]
When rescanning subscrptions for downloads with a locally built docker image from the master branch I'm seeing the following error in the logs:

```
[2021-12-05 18:41:10,951: ERROR/ForkPoolWorker-8] Task home.tasks.check_reindex[a66de92d-d3ba-4329-972e-768e4a831d73] raised unexpected: KeyError('hits')
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 118, in check_reindex
    reindex_old_documents()
  File "/app/home/src/reindex.py", line 583, in reindex_old_documents
    reindex_handler.check_outdated()
  File "/app/home/src/reindex.py", line 157, in check_outdated
    self.all_playlist_ids = self.get_outdated_playlists(playlist_daily)
  File "/app/home/src/reindex.py", line 149, in get_outdated_playlists
    all_playlist_ids = [i["_id"] for i in response_dict["hits"]["hits"]]
KeyError: 'hits'
```

It does seem to complete the rescan despite this.

**Comment by bbilly1** [comment]
Oh good catch! I assume you currently don't have any playlists indexed? Need to handle an empty index there. Thanks for pointing it out!

Btw: Maybe you are wondering, but the task `check_reindex` is responsible to reindex old videos, channels and playlists, that's currently unceremoniously triggered after the `update_subscribed` aka *rescan* task completes. So yes, it completes the rescan but it fails the reindex. Hope to be able to flexible trigger them on an schedule in an upcoming version.

Thanks for your input!

**Comment by bbilly1** [comment]
Is now merged in v0.0.9. Thanks for your help.
